addons/website_forum/static/src/js/website_forum.js

```diff
@@ -317,13 +320,19 @@ publicWidget.registry.websiteForum = publicWidget.Widget.extend({
             } else if (data.success) {
                 var elem = $link;
                 if (data.success === 'post_flagged_moderator') {
-                    elem.data('href') && elem.html(' Flagged');
-                    var c = parseInt($('#count_flagged_posts').html(), 10);
-                    c++;
-                    $('#count_flagged_posts').html(c);
+                    let $child = elem.children(),
+                        $count_flagged_posts = $('span#count_flagged_posts'),
+                        c = parseInt($count_flagged_posts.html(), 10) + 1;
+
+                    elem.html(' Flagged').prepend($child);
+                    $count_flagged_posts.removeClass("badge-light");
+                    $count_flagged_posts.addClass("badge-danger");
+                    $count_flagged_posts.html(c);
+                    elem.next('#flag_validator').removeClass('d-none');
                 } else if (data.success === 'post_flagged_non_moderator') {
-                    elem.data('href') && elem.html(' Flagged');
-                    var forumAnswer = elem.closest('.forum_answer');
+                    let $child = elem.children(),
+                        forumAnswer = elem.closest('.forum_answer');
+                    elem.html(' Flagged').prepend($child);
```

Comment:
:cry: this is not translated.. I know it was there before but :/
maybe just add a TODO for now, I didn't yet review in depth the business code, so we will tackle this one in next review